File: Python/OOP/Calculate sum sub mul div of two numbers.py
class Calculate():
    def __init__(self):

        a = int(input("Input value of a: "))
        b = int(input("Input value of b: "))
        
        self.a = a
        self.b = b

    def Sum(self):
        self.Sum = self.a + self.b
        return self.Sum

    def Sub(self):
        self.Sub = self.a - self.b
        return self.Sub

    def Mul(self):
        self.Mul = self.a * self.b
        return self.Mul

    def Division(self):
        self.Divison = (self.a)/(self.b)       
        return self.Divison

result = Calculate()

print("The sum of two numbers is: ", result.Sum())
print("The substract of two numbers is: ", result.Sub())
print("The multiple of two numbers is: ", result.Mul())
print("The division of two numbers is: ", result.Division())

File: C++/Variables_and_Constants/Calculate sum sub mul div of two numbers.cpp
#include<iostream>

int main()
{
	float a, b;
	
	std::cout << "Input value of a: ";
	std::cin >> a;
	
	std::cout << "Input value of b: ";
	std::cin >> b;
	
	float sum = a + b;
	float sub = a - b;
	float mul = a*b;
	float divide = a/b;
	
	std::cout << "------------------------------"<< std::endl;
	std::cout << "Value of sum is: " << sum << std::endl;
	std::cout << "Value of sub is: " << sub << std::endl;
	std::cout << "Value of mul is: " << mul << std::endl;
	std::cout << "Value of divide is: " << divide << std::endl;
}